Makefile

```diff
@@ -206,53 +213,37 @@ dist_tarballs: docker
 	docker rm tikv-binary-extraction-dummy
 
 # Create tags of the docker images
-.PHONY: docker-tag
 docker-tag: docker-tag-with-git-hash docker-tag-with-git-tag
 
 # Tag docker images with the git hash
-.PHONY: docker-tag-with-git-hash
 docker-tag-with-git-hash:
 	docker tag pingcap/tikv pingcap/tikv:${TIKV_BUILD_GIT_HASH}
 
 # Tag docker images with the git tag
-.PHONY: docker-tag-with-git-tag
 docker-tag-with-git-tag:
 	docker tag pingcap/tikv pingcap/tikv:${TIKV_BUILD_GIT_TAG}
 
+
+## Execution environment
+## -------
+
+export DYLD_LIBRARY_PATH
```

Comment:
Why re-exporting it? Isn't it exported already?

Reply:
you are right